Testing/functional/tests/cypress/integration/e2e/timeline/bcCancerScreening.js

```diff
@@ -31,46 +66,17 @@ describe("Diagnostic Imaging", () => {
             });
     });
 
-    it("Validate file download", () => {
+    it("Validate result file download", () => {
         cy.get("[data-testid=timelineCard")
+            .filter(`:contains(BC Cancer Result)`)
             .first()
-            .within(() => {
-                cy.get("[data-testid=bccancerscreeningTitle]")
-                    .should("be.visible")
-                    .click({ force: true });
-                cy.get(
-                    "[data-testid=bc-cancer-screening-download-button]"
-                ).should("be.visible");
-                // Test for generic message modal regarding sensitive data
-                cy.get(
-                    "[data-testid=bc-cancer-screening-download-button]"
-                ).click();
-                cy.document()
-                    .find("[data-testid=generic-message-modal]")
-                    .should("be.visible");
-                // Submit the generic message modal, which should close the modal
-                cy.document()
-                    .find("[data-testid=generic-message-submit-btn]")
-                    .click();
-                cy.document()
-                    .find("[data-testid=generic-message-modal]")
-                    .should("not.exist");
-
-                // Test for file download with the entry date suffix.
-                cy.get("[data-testid=entryCardDate]")
-                    .first()
-                    .invoke("text")
-                    .then((text) => {
-                        var date = new Date(text);
-                        var dateSuffix = date
-                            .toISOString()
-                            .slice(0, 10)
-                            .replace(/-/g, "_");
+            .within(() => downloadFileTests("bc_cancer_result"));
+    });
 
-                        cy.verifyDownload(`bc_cancer_screening_${dateSuffix}`, {
-                            contains: true,
-                        });
-                    });
-            });
+    it("Validate recall file download", () => {
+        cy.get("[data-testid=timelineCard]")
+            .filter(`:contains(BC Cancer Screening)`)
```

Comment:
```suggestion
            .filter(`:contains("BC Cancer Screening")`)
```

It feels off without quotes in there.  I assume it functions the same.